### DISCUSSION TITLE
many m4s in django media directory

### DISCUSSION THREAD

**Discussion by kasir-barati** [aminyazdanpanah/python-ffmpeg-video-streaming#92] [Ideas]
Hey @aminyazdanpanah 

I hope you're doing well, I was just wondered why you did not mentioned that this `dash.output` generates too much `m4s` file
![image](https://user-images.githubusercontent.com/73785723/177882289-268531b5-c46a-40ed-9069-da92f165ceac.png)
which is normal and maybe it is better to use specific directory for each video to keep things more structured and organized. BTW TBH I am worried about inode and what if we wanted to upload many videos. Obviously saving in FS is not a wise and rational decision AFAIK. BTW I guess I should try to use MongoDB GridFS in conjunction DASH.

But I really appreciate your work and I like to connect to you in LinkedIn if possible. Check my GitHub profile and please send me a connection request.

**Comment by aminyazdanpanah** [comment]
It is how dash works! MPEG-DASH works by breaking the content into a sequence of small segments and m4s files are the small segments. See [dash Doc](https://dashif.org/) and [its wiki](https://en.wikipedia.org/wiki/Dynamic_Adaptive_Streaming_over_HTTP) for more information. You can separate streams by making a new directory and creating dash files in that directory. for example:

``` python
dash.output('/var/media/FIRST-STREAM-/STREAM-NAME.mpd')
dash.output('/var/media/SECOND-STREAM-/STREAM-NAME.mpd')
dash.output('/var/media/THIRD-STREAM-/STREAM-NAME.mpd')
...
```
